core/templates/pages/skill-editor-page/editor-tab/skill-misconceptions-editor/misconception-editor.component.spec.ts

```diff
@@ -116,7 +117,7 @@ describe('Misconception Editor Component', () => {
 
   it('should save name when clicking on save button', () => {
     let updateNameSpy = spyOn(
-      skillUpdateService, 'updateMisconceptionName').and.returnValue(null);
+      skillUpdateService, 'updateMisconceptionName').and.returnValue();
```

Comment:
for Callthrough we need to define all values that are used inside updateMisconceptionName. As Callthrough, spy whole updateMisconceptionName function and then return value.  But here we just want unit testing of "proper working of save button". So end result of updateMisconceptionName function is enough.

Thats why i avoid Callthough many times.

Reply:
Please change it wherever it can be done without a lot of changes, there are still some instances left in the PR. Using `returnValue()` doesn't really make any sense, we should use dedicated functions for these cases as a good coding practice.

Reply:
Done